### PR TITLE
Docs follow up

### PR DESCRIPTION
**Pull request by chriskim06**
Here are the rest of the changes. There were a couple things I missed in my previous PRs and a couple changes based on the review in https://github.com/kubernetes-sigs/krew/pull/690

### CODE REVIEW

File: site/content/docs/developer-guide/plugin-manifest.md [hunk 1]
```diff
@@ -95,8 +95,8 @@ accessible to download from a user’s machine. The relevant fields are:
 ## Specifying platform-specific instructions
 
 Krew makes it possible to install the same plugin on different operating systems
-(e.g., Windows, macOS, and Linux) and different architectures
-(e.g., `amd64`, `386`, and `arm`).
+(e.g., `windows`, `darwin` (macOS), and `linux`) and different architectures
```
Comment (corneliusweig) [inline site/content/docs/developer-guide/plugin-manifest.md:98]:
```suggestion
(e.g., `windows`, `darwin`/macOS, and `linux`) and different architectures
```
Nested parentheses in text look wonky.

**Review by corneliusweig — COMMENTED** [review]
/lgtm
/approve
/hold In case you want to avoid nested parentheses in text.

File: site/content/docs/developer-guide/plugin-manifest.md (hunk 1 above)
Comment (chriskim06) [reply site/content/docs/developer-guide/plugin-manifest.md:98]: hmm I agree, I think the suggestion also looks a little wonky as well though. this is how it was before though with the nested parentheses. @ahmetb @corneliusweig do either of you have a preference?

**Comment by chriskim06** [comment]
I'm going to leave it as is for now, especially since this is how it was before.

/hold cancel

**Pull request merged**